### PR TITLE
Puerto Rico–based chains

### PR DESCRIPTION
**Pull request by 1ec5**
Added several major chains based in Puerto Rico. Some of these chains also have locations elsewhere in the U.S. or elsewhere in the Caribbean.

### CODE REVIEW

File: brands/amenity/cinema.json [hunk 1]
```diff
@@ -47,6 +47,27 @@
...
+      "ag",
+      "do",
+      "fr",
+      "gy",
+      "kn",
+      "lc",
+      "nl",
+      "pa",
+      "tt",
+      "us"
```
Comment (1ec5) [inline brands/amenity/cinema.json]: Specifically, Puerto Rico (`pr`) and the U.S. Virgin Islands (`vi`).

File: brands/amenity/cinema.json [hunk 2]
```diff
@@ -47,6 +47,27 @@
...
   },
+  "amenity/cinema|Caribbean Cinemas": {
+    "countryCodes": [
+      "ag",
+      "do",
+      "fr",
+      "gy",
+      "kn",
+      "lc",
+      "nl",
```
Comment (1ec5) [inline brands/amenity/cinema.json]: Specifically, Sint Maarten (`sx`).

File: brands/amenity/cinema.json [hunk 3]
```diff
@@ -47,6 +47,27 @@
       "short_name": "B&B"
     }
   },
+  "amenity/cinema|Caribbean Cinemas": {
+    "countryCodes": [
+      "ag",
+      "do",
+      "fr",
```
Comment (1ec5) [inline brands/amenity/cinema.json]: Specifically, Guadeloupe (`gp`).

File: brands/amenity/bank.json [hunk 4]
```diff
@@ -1278,24 +1278,6 @@
...
-  "amenity/bank|Banco Popular": {
-    "countryCodes": ["us", "vi"],
-    "matchNames": ["popular"],
-    "tags": {
-      "amenity": "bank",
-      "brand": "Banco Popular",
-      "brand:en": "Popular Bank",
-      "brand:es": "Banco Popular",
-      "brand:wikidata": "Q537259",
-      "brand:wikipedia": "es:Banco Popular Español",
```
Comment (1ec5) [inline brands/amenity/bank.json]: Banco Popular Español was a bank in Spain. It did have a sister company in the U.S., but not under that name.

File: brands/amenity/bank.json [hunk 5]
```diff
@@ -3022,8 +3004,39 @@
...
+    "matchNames": [
+      "1st bancorp",
+      "1st bank",
+      "first bancorp"
+    ],
+    "tags": {
+      "amenity": "bank",
+      "brand": "First Bank",
+      "brand:wikidata": "Q5452333",
+      "brand:wikipedia": "en:First BanCorp",
```
Comment (1ec5) [inline brands/amenity/bank.json]: Note the distinction between “[First Bancorp](https://en.wikipedia.org/wiki/First_Bancorp)” and “[First BanCorp](https://en.wikipedia.org/wiki/First_BanCorp)”. 🤕

File: brands/amenity/bank.json [hunk 6]
```diff
@@ -3022,8 +3004,39 @@
...
+    "tags": {
+      "amenity": "bank",
+      "brand": "First Bank",
+      "brand:wikidata": "Q5452332",
+      "brand:wikipedia": "en:First Bancorp",
+      "name": "First Bank"
+    }
+  },
+  "amenity/bank|First Bank~(Puerto Rico)": {
+    "countryCodes": ["us"],
```
Comment (1ec5) [inline brands/amenity/bank.json]: Specifically, Florida, Puerto Rico (`pr`), and the U.S. Virgin Islands (`vi`).

File: brands/amenity/fast_food.json [hunk 7]
```diff
@@ -1176,6 +1176,23 @@
       "takeaway": "yes"
     }
   },
+  "amenity/fast_food|Meson Sandwiches": {
+    "countryCodes": ["us"],
+    "matchNames": [
+      "el meson",
```
Comment (1ec5) [inline brands/amenity/fast_food.json]: Most existing features in Puerto Rico are tagged “El Mesón”, which this line will match.

File: brands/amenity/bank.json (hunk 5 above)
Comment (bhousel) [reply brands/amenity/bank.json]: ouch

Comment (bhousel) [reply brands/amenity/bank.json]: another reason why wikidata qids are better for everything

**Comment by bhousel** [comment]
Looks good, thanks @1ec5  👍

**Pull request merged**